src/collections.ts

```diff
@@ -21,9 +21,10 @@ export class Py_List extends pytypes.Py_Object implements Iterable {
     }
     return list;
   }
-  public len(): Py_Int {
-    return Py_Int.fromInt(this._list.length);
+  public len(): number {
+          return this._list.length;
```

Comment:
Actually, looks like we're currently inconsistent. I usually do 2 spaces per tab, so we might eventually just switch to that. Either way, we don't use tab characters. :)